core/GlobalState.h

```diff
@@ -69,7 +69,7 @@ class GlobalState final {
     SymbolRef enterTypeMember(Loc loc, SymbolRef owner, NameRef name, Variance variance);
     SymbolRef enterTypeArgument(Loc loc, SymbolRef owner, NameRef name, Variance variance);
     SymbolRef enterMethodSymbol(Loc loc, SymbolRef owner, NameRef name);
-    SymbolRef enterNewMethodOverload(Loc loc, SymbolRef original, core::NameRef originalName, u2 num,
+    SymbolRef enterNewMethodOverload(Loc loc, SymbolRef original, core::NameRef originalName, u4 num,
```

Comment:
given than unique names need to be u4 due to https://github.com/sorbet/sorbet/pull/2791/files#r395269511, made this also u4 for consistency